pandas/tests/arrays/test_array.py

```diff
@@ -278,7 +278,7 @@ def test_array_inference_fails(data):
     tm.assert_extension_array_equal(result, expected)
 
 
-@pytest.mark.parametrize("data", [np.array([[1, 2], [3, 4]]), [[1, 2], [3, 4]]])
+@pytest.mark.parametrize("data", [np.array(0)])
```

Comment:
we pass thru 2-D pandas arrays?

Reply:
DTA/TDA._validate_listlike calls pd.array on possibly-2D inputs, which go through PandasArray